ISOLATED_SERVER_setup.md

```diff
@@ -32,9 +32,21 @@ make clean; make
 ```
 <LOOKUP_NODE_MODE>true</LOOKUP_NODE_MODE>
 <ENABLE_SC>true</ENABLE_SC>
-<SCILLA_ROOT>/home/ubuntu/scilla</SCILLA_ROOT>
+<SCILLA_ROOT>scilla</SCILLA_ROOT>
```

Comment:
Let's not touch the default config (also in the doc), esp about Scilla

Reply:
I have raised this many times and not got a clear answer, the master branch of Zilliqa has the value as /scilla in constants.xml this is what is set in evm_support_main now although when I asked in a meeting i was told it is relative to current dir. Which one is correct?

Reply:
This diff (and https://github.com/Zilliqa/Zilliqa/blob/master/ISOLATED_SERVER_setup.md) suggests it was set to /home/ubuntu.  Not sure what's the best instruction, just no reason for this PR to change it.